lib/Rex/Commands/Sysctl.pm

```diff
@@ -79,6 +94,11 @@ sub sysctl {
       Rex::Logger::debug("$key has already value $val");
     }
 
+    if ( exists $options{persistent} && $options{persistent} eq "true" ) {
```

Comment:
I think `sysctl_remove` should only be called if `persistent` is FALSE.

As a somewhat related topic, I would find `ensure => 'present'` and `ensure => 'absent'` more clear about what it means and does, plus more in-line with the usual rex syntax for other things.